src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/clsm_ensupd_enkf_update.F90

```diff
@@ -2738,10 +2738,10 @@ subroutine write_smapL4SMaup( option, date_time, work_path, exp_id, N_ens,    &
                 data_h_9km_tile_8 = real(nodata_generic,kind(0.0D0)) ! init (not in grid2tile!)
                 data_v_9km_tile_8 = real(nodata_generic,kind(0.0D0)) ! init (not in grid2tile!)
 
-                call grid2tile( tile_grid_g, N_catf, tile_coord_f%i_indg, tile_coord_f%j_indg, data_h_9km_grid_8, &
+                call grid2tile( tile_grid_g, N_catf, tile_coord_f%hash_i_indg, tile_coord_f%hash_j_indg, data_h_9km_grid_8, &
                      data_h_9km_tile_8 )
 
-                call grid2tile( tile_grid_g, N_catf, tile_coord_f%i_indg, tile_coord_f%j_indg, data_v_9km_grid_8, &
+                call grid2tile( tile_grid_g, N_catf, tile_coord_f%hash_i_indg, tile_coord_f%hash_j_indg, data_v_9km_grid_8, &
```

Comment:
Same comment as above.

Reply:
I feel we should not cheery-picking but be consistent through out the codes: hash_x_indg  is used for computation and x_indg is used to keep the original record whenever we can. The only place we should use X_indg probably is here https://github.com/GEOS-ESM/GEOSldas/blob/b674b877cac911106b55b5f3389196fa9cc9d1c3/src/Components/GEOSldas_GridComp/GEOSmetforce_GridComp/LDAS_Forcing.F90#L4674-L4679

Reply:
This isn't cherry-picking.  For a grid2tile operation, we need to use the i,j indices that are associated with grid that is involved in the operation.  Here, the grid is the original grid associated with the tile space, not the "hash" grid used in the perturbations and to support the EnKF analysis.   I know this is a bit subtle because it looks like it's part of the EnKF analysis, but it's really just a complicated special case to __output__ (1d array) tile-space fields in gridded (2d array) format (taking advantage of the fact that by construction the EASE-based tile space has at most one tile in each grid cell.) 
Again, the grid involved here is NOT the hash grid that is used for perturbations or to support the EnKF analysis.

Reply:
It seems it is just part of SMAP writing. Changed